src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1150,6 +1150,13 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
 
     set_arguments();
     GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_IF(debug_config->list_layers == 1) {
```

Comment:
Is there a reason doing this in network? Could we do this in program?

Reply:
No special reason, I don't care for it. Could you let me know where's your recommendation in program? then I'll update it.

Reply:
Hmm not important to me too, sorry for the confusion :) 
BTW can this cover body networks in loop?

Reply:
I tried to test it with loop model with unrolling, but I have an issue loop in model.
I changed some code because I heard some condition run loop with unrolling.
But there's some issue run loop in my side. (I'm not familiar loop model)
I found only 3 models with loop type( grep type="Loop" in IR) from all models in 1 team covered.
yolo_v3 -> always unrolling
deepspeech2 -> running failed
ocr-perpetuuiti -> hang issue
**Code change for force enable unrolling loop**
![image](https://user-images.githubusercontent.com/20204153/236123238-c96396b3-63c8-4ee4-9093-9752100c3e30.png)

So my question, 
1. Code change is correct for force running loop?
2. If true, do we have an known issue for loop? or If not could you share the model which is working well with loop?

Reply:
You can try with text-recognition-0011 for running a network w/ loops. 
With that, you don't need any change in transformations_pipeline.cpp to prevent unrolling.

Reply:
Update to show loop body primitives.